Add Shaker (#355)
godotengine/awesome-godot: README.md

- [Maaack's Scene Loader](https://github.com/Maaack/Godot-Scene-Loader) - Scene loader that includes a loading screen, progress bar, and error handling.
- [PixelPen](https://github.com/pixelpen-dev/pixelpen) - Pixel art drawing and animation: Common drawing tools such as selection, shape creation, mirroring, tiling, and animation features within the Godot editor.

> - [TerraBrush](https://github.com/spimort/TerraBrush) - Minimal Terrain heightmap editor that features sculpting, painting textures, LOD, and MultiZones.
> - [Shaker](https://github.com/Eneskp3441/Shaker) - Plugin that adds shaking and emitters for cameras, nodes or any property in 2D and 3D. 

Reviewer: ```suggestion
- [Shaker](https://github.com/Eneskp3441/Shaker) - Plugin that adds shaking and emitters for cameras, nodes or any property in 2D and 3D.
- [TerraBrush](https://github.com/spimort/TerraBrush) - Minimal Terrain heightmap editor that features sculpting, painting textures, LOD, and MultiZones.
```

> - [Shaker](https://github.com/Eneskp3441/Shaker) - A plugin that makes it easy for you to add shake and Emitter for Camera/Node/Parameters for 2D and 3D. 

Reviewer: ```suggestion
- [Shaker](https://github.com/Eneskp3441/Shaker) - Plugin that adds shake and emitters for cameras, nodes or any property in 2D and 3D. 
```
